### PR TITLE
update the css to match dartlang.org

### PR DESCRIPTION
**Pull request by devoncarew**
- update the css to match dartlang.org
- some work towards https://github.com/dart-lang/dartdoc/issues/1372
- done as two commits to make review easier

This does not change page layout of the top-of-page navigation.

@jcollins-g 
screen shot 2017-05-12 at 3 16 16 am

### CODE REVIEW

File: lib/src/html/template_data.dart [hunk 1]
```diff
@@ -32,7 +32,10 @@ abstract class TemplateData<T extends Documentable> {
   String get title;
   String get layoutTitle;
   String get metaDescription;
+
   List get navLinks;
+  Documentable get navLinksLast => navLinks.last;
```
Comment (jcollins-g) [inline lib/src/html/template_data.dart]: Might we instead make name and kind getters from the templates and let the template classes define how we get them?

File: testing/test_package_docs/ex/Deprecated/operator_equals.html [hunk 2]
```diff
@@ -66,10 +66,7 @@ <h1 class="title">
   <div class="row">
 
   <div class="col-xs-6 col-sm-3 col-md-2 sidebar sidebar-offcanvas-left">
-    <h5><a href="index.html">test_package</a></h5>
-    <h5><a href="ex/ex-library.html">ex</a></h5>
-    <h5><a href="ex/Deprecated-class.html">Deprecated</a></h5>
-
+    <h5>class Deprecated</h5>
```
Comment (jcollins-g) [inline testing/test_package_docs/ex/Deprecated/operator_equals.html]: Was it a conscious choice to drop the link to class even though this is a member page?

Comment (devoncarew) [inline testing/test_package_docs/ex/Deprecated/operator_equals.html]: Discussed off-line - this was part of simplifying the LHS navigation.

File: lib/src/html/template_data.dart (hunk 1 above)
Comment (devoncarew) [inline lib/src/html/template_data.dart]: Renamed to `parent`, and added some more semantic accessors on `TemplateData` (`name` and `kind`).

**Comment by devoncarew** [comment]
ptal, the new diff is pretty small

**Pull request merged**